tests/config/test_validation.py

```diff
@@ -15,26 +17,57 @@ def invalid_settings_file(tmp_path, toml_content):
 @pytest.mark.parametrize("toml_content", ["invalid toml["])
 def test_update_failure_invalid_toml(invalid_settings_file):
     with pytest.raises(dynaconf.vendor.toml.decoder.TomlDecodeError):
-        update_settings_files([invalid_settings_file.absolute()])
+        update_settings_files(invalid_settings_file.absolute())
 
 
 @pytest.mark.parametrize("toml_content", [""])
 def test_update_failure_invalid_preliminary_config(invalid_settings_file):
+    old_value = os.environ.get("SECRETS_FOR_DYNACONF", "")
+    os.environ["SECRETS_FOR_DYNACONF"] = ""
+
     with pytest.raises(dynaconf.validator.ValidationError):
-        update_settings_files([invalid_settings_file.absolute()])
+        update_settings_files(invalid_settings_file.absolute())
+
+    os.environ["SECRETS_FOR_DYNACONF"] = old_value
 
 
 @pytest.mark.parametrize(
     "invalid_toml,pattern_in_exception",
     [
-        ["config_with_strategy.toml", "publisher.*.active"],
+        ["config_with_invalid_strategy.toml", "break_between_events_in_minutes"],
+        ["config_with_invalid_mobilizon.toml", "mobilizon"],
         ["config_with_preliminary.toml", "publishing.window.begin"],
+    ],
+)
+def test_update_failure_config_base_validators(invalid_toml, pattern_in_exception):
+
+    with pytest.raises(dynaconf.validator.ValidationError) as e:
+        update_settings_files(
+            pkg_resources.resource_filename("tests.resources.config", invalid_toml)
+        )
+    assert e.match(pattern_in_exception)
+
+
+@pytest.mark.parametrize(
+    "invalid_toml,pattern_in_exception",
+    [
+        ["empty.toml", "publisher.*.active"],
         ["config_with_invalid_telegram.toml", "token"],
     ],
 )
-def test_update_failure_config_without_publishers(invalid_toml, pattern_in_exception):
+def test_update_failure_config_all_validators(invalid_toml, pattern_in_exception):
+
+    old_value = os.environ.get("SECRETS_FOR_DYNACONF", "")
```

Comment:
are you sure it's necessary to restore the old value? I think pytest reloads the env at every test

Reply:
I'm not sure why but if I don't the next tests fail. I believe It could've Something to do with parallel test execution